db1/fakerData.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable prefer-template */
+/* eslint-disable no-plusplus */
+/**
+ * Fake data for:
+ *  users table:
+ *    -id: starting at 1 in increasing order
+ *    -names: first and last
+ *  reviews table:
+ *    -id: start at 1 in increasing order
+ *    -ratings: out of 5?
+ *    -title: Blurb of review i.e. subject in email
+ *    -description: actual review
+ *    -report abuse: 0 or 1 for true or false.
+ *    -isProductPropGood1Good: 0 or 1 for true or false (correlates to product table's prop1)
+ *    -isProductPropGood2Good: 0 or 1 for true or false (correlates to product table's prop2)
+ *    -isProductPropGood3Good: 0 or 1 for true or false (correlates to product table's prop3)
+ *    -created_on: date record updated/created?
+ *    -userId: correlates to an id on user table's id
+ *    -productId: correlates to an id on product table's id
+ *  ReviewFeedbacks table;
+ *    -id: start @ 1
+ *    -reviewId: refers to id on reviews table;
+ *    -userId: refers to id on user table
+ *    -isHelpful: 0 or 1 for true/false
+ *  Products table;
+ *    -id: start @ 1
+ *    -name: name of product;
+ *    -productCondition: either new or used;
+ *    -seller: kinda random names could do first+last or company names;
+ *    -prop1: A characteristic about the product, she cycles between 2 different one for this column
+ *    -prop2: A characteristic about the product, she cycles between 2 different one for this column
+ *    -prop3: A characteristic about the product, she cycles between 2 different one for this column
+ */
+const Faker = require('faker');
+
+const goalRecords = 1000000;
+
+const usersTable = () => {
+  let usersArray = '';
+  // shouldn't matter we start id @ 0 instead of 1;
+  for (let i = 0; i < goalRecords; i++) {
+    usersArray += `${i.toString()},`;
+    usersArray += (`"${Faker.name.firstName()} ${Faker.name.lastName()}"`);
```

Comment:
I think that in CSV if you want to to create strings you need to double-quote it, otherwise it won't show up with quotes around it in the CSV. But it is probably more preformant to switch it from .toString() to "".